make.ps1

```diff
@@ -170,9 +155,9 @@ function Docs-Command
 	}
 
 	./make.ps1 version
-	Invoke-Expression "$utilityPath $modID --docs | Out-File -Encoding 'UTF8' DOCUMENTATION.md"
-	Invoke-Expression "$utilityPath $modID --weapon-docs | Out-File -Encoding "UTF8" WEAPONS.md"
-	Invoke-Expression "$utilityPath $modID --lua-docs | Out-File -Encoding 'UTF8' Lua-API.md"
+	InvokeCommand "$utilityPath $modID --docs | Out-File -Encoding 'UTF8' DOCUMENTATION.md"
+	InvokeCommand "$utilityPath $modID --weapon-docs | Out-File -Encoding "UTF8" WEAPONS.md"
```

Comment:
Looks like this was broken all along. We put `""` around UTF8 while we already were inside a string!

Reply:
This raises a second question: it looks like we have never supported a `docs` target in the real makefile, and we should make sure that the same options are exposed in both.

Should we remove this one and make sure that people know to run the utility commands directly on all platforms, or update this and add a matching target to the makefile? The fact that this has apparently been broken here and in the engine for years(?) without anybody noticing suggests that removing it would be better.